src/App.vue

```diff
@@ -282,18 +312,11 @@ export default class App extends Mixins(mixins.TransactionMixin, NodeErrorMixin)
       tmaSdkService.init(data?.TG_BOT_URL);
 
       await this.setApiKeys(data?.API_KEYS);
-      await this.setEthBridgeSettings(data.ETH_BRIDGE);
+      this.setEthBridgeSettings(data.ETH_BRIDGE);
       this.setFeatureFlags(data?.FEATURE_FLAGS);
       this.setSoraNetwork(data.NETWORK_TYPE);
       this.setEvmNetworksApp(data.EVM_NETWORKS_IDS);
       this.setSubNetworkApps(data.SUB_NETWORKS);
-
-      this.subscribeOnExchangeRatesApi();
-
-      if (data.PARACHAIN_IDS) {
-        api.bridgeProxy.sub.parachainIds = data.PARACHAIN_IDS;
```

Comment:
why was it removed?

Reply:
As it turns out, we have no reason to replace id on testnets. I added it for these purposes, but this is unused code.